docker-compose/cassandra-3.11/docker-compose-dev-mode.yml

```diff
@@ -8,7 +8,6 @@ services:
     ports:
       - "9042:9042"
       - "8081:8081"
-      - "8084:8084"
```

Comment:
So this is the... metrics endpoint in Coordinator? And one we don't really need to expose.

Reply:
something else is using this port when running in the GitHub Actions environment, no idea what. So yeah I just removed it and moved on.